Documentation/network/lb-ipam.rst

```diff
@@ -50,11 +53,26 @@ After adding the pool to the cluster, it appears like so.
     NAME        DISABLED   CONFLICTING   IPS AVAILABLE   AGE
     blue-pool   false      False         65788           2s
 
-.. note::
-    The amount of available IPs in the pool is lower than the actual sum of all
-    usable IPs in the CIDRs because the allocation logic is limited to 65536 IPs
-    per CIDR. CIDRs containing more than 65536 IPs can be broken down into multiple
-    smaller CIDRs to achieve full utilization.
+CIDRs, Ranges and dodgy IPs
+---------------------------
+
+An IP pool can have multiple blocks of IPs. A block can be specified with CIDR
+notation (<prefix>/<bits>) or a range notation with a start and stop IP. As
+pictured above.
+
+CIDRs are often used to specify routable IP ranges. By convention, the first
+IP of a range and the last IP of a range are reserved. The first IP is the 
```

Comment:
```suggestion
and the last IP of a CIDR are reserved. The first IP is the 
```
Using the word "range" here can be a bit confusing, as this section describes a concept of range that's an alternative to a CIDR.